admin/load.php

```diff
@@ -531,6 +534,29 @@ function perflab_enqueue_modules_page_scripts() {
 	wp_enqueue_style( 'thickbox' );
 
 	wp_enqueue_script( 'plugin-install' );
+
+	wp_enqueue_script(
+		'perflab-admin',
+		plugin_dir_url( __FILE__ ) . 'perflab-admin.js',
```

Comment:
Let's name this script more specific to what it is: It's not a general script for the admin screen, it's only for the very specific scenario of migrating modules to their standalone plugins. Maybe `perflab-module-migration-notice`?